xml/System/Double.xml

```diff
@@ -1815,6 +1815,7 @@
       <Docs>
         <summary>Converts the string representation of a number to its double-precision floating-point number equivalent.</summary>
         <altmember cref="Overload:System.Double.TryParse" />
+        <remarks>In .NET Core 3.0 and above, parsing will attempt to parse the input as if to infinite precision, and then round to the nearest representation as per IEEE 754. This behavior is different and not backwards compatible with previous versions, where attempting to parse a given value that was too large to represent failed.</remarks>
```

Comment:
Since .NET Core 2.0 also attempted to "parse the input as if to infinite precision, and then round to the nearest representation" and just didn't correctly handle "overflow", it might be clearer to just say:
```suggestion
        <remarks>In .NET Core 3.0 and above, values that are too large to represent will be rounded to <xref:System.Double.PositiveInfinity/> or <xref:System.Double.NegativeInfinity/> as required by the IEEE 754 specification. In prior versions, including .NET Framework, parsing values that are too large to represent would result in failure.</remarks>
```

Reply:
```suggestion
        <remarks>In .NET Core 3.0 and later, values that are too large to represent are rounded to <xref:System.Double.PositiveInfinity> or <xref:System.Double.NegativeInfinity> as required by the IEEE 754 specification. In prior versions, including .NET Framework, parsing a value that was too large to represent resulted in failure.</remarks>
```

Some tweaks to Tanner's suggestion.

Reply:
Hopefully this has been resolved - I applied @gewarren's modified tweaks to @tannergooding's suggestion, but closed off the System.[Double/Single].[Positive/Negative]Infinity tags with `/` for XML and `%2a` for markdown.

Reply:
The <xref> links should not have a closing `/` - you can see that it's already working here: https://docs.microsoft.com/en-us/dotnet/api/system.double?view=netframework-4.8#remarks.